testutil/e2e/paymentE2E.go

```diff
@@ -146,7 +149,7 @@ func (lt *lavaTest) checkPayment(providers []string, startBalances []sdk.Coin) {
 	}
 
 	// get new balance and checks that at least one provider's balance was increased
-	newBalances, err := lt.getBalances(providers)
+	newBalances, err := lt.getRewards(providers)
```

Comment:
change the variable name for clarity. Those are rewards, not balances

Reply:
done